_schemaV2.graphql

```diff
@@ -317,7 +317,6 @@ type Alert {
   ): String!
```

Comment:
The removal of fields and connection types here looks ✅ but I always get a little nervous of the MP magic.
But to chat out loud, the removal of this stuff here is OK since Volt was the only client using these fields and Volt not longer is using them